### ISSUE TITLE
Model training is unstable

### ISSUE THREAD

**Issue by cjcjcj666** [sadrasafa/StereoGS#3]
Training progress:  23%|███▎          | 7000/30000 [05:55<25:15, 15.18it/s, Loss=0.0982154, Stereo Loss=0.0000000]/root/miniconda3/envs/gaussian_splatting/lib/python3.7/site-packages/torch/functional.py:478: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/TensorShape.cpp:2894.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Training progress:  27%|███▏        | 8080/30000 [37:14<9:40:37,  1.59s/it, Loss=0.0666318, Stereo Loss=0.0108869]

![Image](https://github.com/user-attachments/assets/be0722c6-3085-4e72-94fb-70b8334e9d50)

Hello, esteemed author.When I train my own dataset, this problem occurs, what's wrong with the iteration number of 7k iterations is still good and suddenly it does not work, this situation is not a gradient explosion, is there any solution?

**Comment by sadrasafa** [comment]
Hi, thank you for your interest in our work. This is actually not a problem, and it is expected. After the 7k iteration, the stereo network starts to estimate the depth for the image pairs, which is why the training becomes slower. You can ignore the warning message.
